unified-scheduler-pool/src/lib.rs

```diff
@@ -593,25 +1025,25 @@ mod tests {
                 }
                 overall_timings.accumulate(&timings);
             }
-            *self.0.result_with_timings.lock().unwrap() = (overall_result, overall_timings);
+            *self.0.lock().unwrap() = (overall_result, overall_timings);
         }
     }
 
     impl<const TRIGGER_RACE_CONDITION: bool> InstalledScheduler
         for AsyncScheduler<TRIGGER_RACE_CONDITION>
     {
         fn id(&self) -> SchedulerId {
-            self.0.id()
+            todo!();
```

Comment:
probably better to use `unimplemented!()`, I don't believe you are intending to do this, right?

Reply:
you're right: afd580e8f86634447deaa86282e24bd257fbdee5